playbooks/roles/githubsshkeys/tasks/main.yml

```diff
@@ -1,13 +1,16 @@
-- name: install httplib2
-  pip: name=httplib2
+- name: install httplib2 and pyOpenSSL
```

Comment:
I wonder if it might be safer to `pip install` these into a venv to be sure there are no version conflicts with  pyOpenSSL.  That package can be problematic.

Reply:
@bryanlandia thanks for your review. that's great point. about the `pyOpenSSL ` version recently for `ADN` i used this role and it didn't without `pyOpenSSL==16.2.0`

Reply:
@amirtds I'm not sure I understand your comment here.

Reply:
sorry @bryanlandia for the delay. yes i mean on recent Azure spin up that i did it worked with `pyOpenSSL==16.2.0` version. im agree with installing the `pyOpenSSL` via pip to don't have any trouble related to version conflicts;